client/src/pages/HopscotchTour.js

```diff
@@ -119,14 +119,14 @@ const superUserTour = (currentUser, history) => {
         fixedElement: true,
         multipage: true,
         onNext: () =>
-          history.push(Organization.pathFor(currentUser.position.organization))
+          navigate.push(Organization.pathFor(currentUser.position.organization))
```

Comment:
The HopscotchTour hasn't been fully functional for quite some time now (since the switch to flexbox layout, basically), and the component used for it is unmaintained (see #3018).